### PR TITLE
feat(docs): Loading code for composition

### PR DESCRIPTION
**Pull request by XingXiaoWu**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: website/docs/en-US/loading.md [hunk 1]
```diff
@@ -167,6 +237,44 @@ Show a full screen animation while loading data.
...
+      const fullscreenLoading = ref(false);
+      const openFullScreen1 = () => {
+        fullscreenLoading.value = true;
+        setTimeout(() => {
+          fullscreenLoading.value = false;
+        }, 2000);
+      };
+
+      const openFullScreen2 = () => {
+        const loading = proxy.$loading({
```
Comment (iamkun) [inline website/docs/en-US/loading.md:257]: can we use 
```
import { ElLoading } from 'element-plus';
```
here, pls?

Comment (XingXiaoWu) [reply website/docs/en-US/loading.md:257]: It's so embarrassing. I was trying to fix it, but I didn't notice it. I will pay more attention to this part in the subsequent code modification

Comment (iamkun) [reply website/docs/en-US/loading.md:257]: That's not a problem, take your time. Thanks for your lovely contribution.

Comment (XingXiaoWu) [reply website/docs/en-US/loading.md]: have updated

**Review by iamkun — APPROVED** [review]
LGTM

**Pull request merged**